config/blazer.yml

```diff
@@ -0,0 +1,73 @@
+# see https://github.com/ankane/blazer for more info
+
+data_sources:
+  main:
+    url: <%= ENV["BLAZER_DATABASE_URL"] %>
+
+    # statement timeout, in seconds
+    # none by default
+    timeout: 60
+
+    # caching settings
+    # can greatly improve speed
+    # off by default
+    cache:
+      mode: slow # or all
+      expires_in: 60 # min
+      slow_threshold: 15 # sec, only used in slow mode
+
+    # wrap queries in a transaction for safety
+    # not necessary if you use a read-only user
+    # true by default
+    # use_transaction: false
+
+    smart_variables:
+      # zone_id: "SELECT id, name FROM zones ORDER BY name ASC"
+      # period: ["day", "week", "month"]
+      # status: {0: "Active", 1: "Archived"}
+
+    linked_columns:
+      # user_id: "/admin/users/{value}"
+
+    smart_columns:
+      # user_id: "SELECT id, name FROM users WHERE id IN {value}"
+
+# create audits
+audit: true
```

Comment:
Since we're using a read-only user, I assume we won't be able to make updates, so do we need audits? I have no problem if we keep them, I just wonder if it's worth filling up that table with non-destructive operations.

Reply:
I think it is still a good idea bc if we run into performance trouble we can always check to see if someone ran a problematic query. Also might be good to get an idea of what information people are accessing most